index.js

```diff
@@ -220,15 +225,7 @@ function isRequestOriginAllowed (reqOrigin, allowedOrigin) {
   }
 }
 
-const _fastifyCors = fp(fastifyCors, {
+module.exports = fp(fastifyCors, {
   fastify: '4.x',
   name: '@fastify/cors'
 })
-
-/**
- * These export configurations enable JS and TS developers
- * to consumer fastify in whatever way best suits their needs.
- */
-module.exports = _fastifyCors
```

Comment:
Please keep the export below.

Reply:
my bad didn't even notice it, good catch! ty